PublicSDKAPITester/RevenueCatAPI.h

```diff
@@ -14,6 +14,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface RevenueCatAPI<RCPurchasesDelegate> : NSObject
 
 + (void)allTheThings;
```

Comment:
perhaps we should rename this one since it doesn't actually cover all the things... 
How about `methodsAndPropertiesInPurchases`?

Reply:
I have a change queued up for this too... ha

Reply:
https://github.com/RevenueCat/purchases-ios/pull/595